dace/codegen/targets/xilinx.py

```diff
@@ -483,6 +505,11 @@ def generate_kernel_boilerplate_pre(self, sdfg, state_id, kernel_name,
                 "#pragma HLS INTERFACE s_axilite port={} bundle=control".format(
                     var_name))
 
+        for arg in stream_args:
+            var_name = re.findall(r"\w+", arg)[-1]
+            kernel_stream.write(
+                "#pragma HLS INTERFACE axis port={}".format(var_name))
```

Comment:
Ugh... Can't we pull these out when we build `stream_args`?

Reply:
As in emit them during building the `stream_args`? Or as in be omitted? For the simple examples, they can be omitted. But this might change with more complex designs, where the HLS pattern matching might break down?

Reply:
As in just save the names in a list when we are constructing `stream_args`, so we don't have to regex them out again afterwards :-)

Reply:
Fixed in https://github.com/carljohnsen/dace/commit/b6be1fc5cb770a3cb187e1ea5bb01ae191806bbb